test_scripts/Smoke/Resumption/ATF_Resumption_3rd_ignition_cycle.lua

```diff
@@ -106,15 +91,31 @@ end
 commonFunctions:newTestCasesGroup("SDL should perform data resumption application is registered within 3 ign cycles")
 
 function Test.IGNITION_OFF()
-  IGNITION_OFF()
+  Test.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+    { reason = "SUSPEND" })
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLPersistenceComplete"):Do(function()
+    Test.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
+      { reason = "IGNITION_OFF" })
+    EXPECT_NOTIFICATION("OnAppInterfaceUnregistered", { reason = "IGNITION_OFF" })
+    EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", { unexpectedDisconnect = false })
+    EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose")
+    SDL:DeleteFile()
+  end)
 end
 
 function Test.Restart_SDL_And_Add_Mobile_Connection()
   Start_SDL_And_Add_Mobile_Connection()
 end
 
 function Test.IGNITION_OFF()
-  IGNITION_OFF()
+  Test.hmiConnection:SendNotification("BasicCommunication.OnExitAllApplications",
```

Comment:
@dboltovskyi Please use `self` instead direct module variable `Test` in member functions.

Reply:
@aderiabin Please check d5a3629